exts/cesium.omniverse/cesium/omniverse/ui/styles.py

```diff
@@ -44,6 +44,15 @@ class CesiumOmniverseUiStyles:
         "Button.Label": {"alignment": Alignment.CENTER_BOTTOM},
     }
 
+    top_bar_button_disabled_style = {
+        "Button": {"padding": 10.0, "stack_direction": Direction.TOP_TO_BOTTOM},
+        "Button.Image": {
+            "alignment": Alignment.CENTER,
+            "color": cl("#808080"),
+        },
+        "Button.Label": {"alignment": Alignment.CENTER_BOTTOM, "color": cl("#808080")},
+    }
+
```

Comment:
Did you check if there is a state selector for this? I believe you should be able to use `Button:disabled` as the selector.